### PR TITLE
Feature: conform to interface spec

### PR DESCRIPTION
**Pull request by ynfle**
To be reviewed before #6

### CODE REVIEW

File: bin/generate.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+set -e
```
Comment (ErikSchierboom) [inline bin/generate.sh:1]: We've recently [updated the spec](https://github.com/exercism/automated-analysis/blob/master/docs/docker.md#representers) to have this file be named `run.sh`.

Comment (ErikSchierboom) [reply bin/generate.sh:1]: I don't know if I can see this somewhere in GitHub, but the executable bit also has to be set on this file.

File: bin/generate.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+if [ -z "$1" ] || [ -z "$2" ]; then
```
Comment (ErikSchierboom) [inline bin/generate.sh:3]: The representer [takes three arguments](https://github.com/exercism/automated-analysis/blob/master/docs/representers/interface.md):

- The slug of the exercise (e.g. two-fer).
- A path to a directory containing the submitted file(s) (with a trailing slash).
- A path to an output directory (with a trailing slash). This directory is writable.

**Review by ErikSchierboom — COMMENTED** [review]
Some small comments!

File: bin/generate.sh (hunk 2 above)
Comment (ErikSchierboom) [reply bin/generate.sh]: Probably not. I assume `joinPath` will do that for you when combining paths.

**Comment by ynfle** [comment]
I will resolve the conflicts and then merge later today

File: bin/generate.sh (hunk 2 above)
Comment (ynfle) [reply bin/generate.sh]: Not sure.
I just use the `/` operator

**Comment by ynfle** [comment]
Just rebase. 
The diff https://github.com/exercism/v3/compare/51c4209..4369edf

**Comment by ynfle** [comment]
Just rebase and a commit message change.
The diff https://github.com/exercism/v3/compare/4369edf..eaa79a6

**Pull request merged**